docs/content/kv_commands.md

```diff
@@ -177,10 +249,12 @@ Takes as an argument a JSON file with a list of keys to delete; for example:
 ]
 ```
 
+The `delete` command also takes an optional `--env` [environment](./environments) argument.
+
 #### Usage
 
 ```sh
-$ wrangler kv:bulk delete f7b02e7fc70443149ac906dd81ec1791 allthethingsdelete.json
+$ wrangler kv:bulk delete --binding=MY_NAMESPACE allthethingsdelete.json
```

Comment:
With this json that I successfully used to bulk upload I got an error

```json
[{
    "key": "five",
    "value": "seven"
},
{
    "key": "six",
    "value": "eight"
}]
```

```console
wrangler kv:bulk delete -b OKV mykeys.json 
Are you sure you want to delete all keys in mykeys.json? [y/n]
y
Error: invalid type: map, expected a string at line 1 column 1
```

Reply:
the deletion json is different--there is an example of this in the docs.